amivapi/events/validation.py

```diff
@@ -114,11 +113,11 @@ def _validate_signup_requirements(self, signup_possible, field, event_id):
 
             # The event has signup, check if it is open
             if not g.get('resource_admin'):
-                now = datetime.now(pytz.utc)
-                if now < event['time_register_start']:
+                now = datetime.utcnow()
```

Comment:
Note: This creates a utc time, but without any timezone info.
Therefore we need to use `.replace(tzinfo=None)` below, as the event times are
utc as well, but *with* timezoneinfo.

Reply:
I think the replace doesn't do anything. utcnow() returns already a naive datetime object with tzinfo None, why would replacing this None with a None do anything? If the db returns things with timezones, then the old code was the correct way to get the current UTC.

Reply:
Ok nevermind the replace is on the event time not on the `now` object.